config/locales/rdr.en.yml

```diff
@@ -32,6 +32,10 @@ en:
     submissions:
       heading: 'Data Submission Form'
       form_error_html: "We encountered an error processing your request. One of our curators will follow up with you shortly. In the meantime, please contact us if you have any questions at <a href='%{email}'>%{email}</a>."
+      instructions_text: 'Thank you for depositing with the Duke Research Data Repository. When you have finished uploading your files, please send a reply to the email you received confirming your submission, or email us at datamanagement@duke.edu. If you have any questions, please review our Submission guidelines or contact us.
+
+
+      Research data curators will review your deposit to help ensure the data meet the submission requirements and are in a format and structure that supports long-term preservation, access, and reuse. If any changes are needed, you will be contacted. Once the processing is complete, the depositor will be notified and provided with a link to their dataset record and persistent identifier. Our aim is to process all deposits in a timely fashion and turnaround is usually one to three business days, however processing time may vary depending on size and type of data.'
```

Comment:
If we're going to include these instructions here in this way, I think it would be better to put them on a single line, wrapped in double quotes, and with `\n` (as many times as needed) to separate the paragraphs.  At least I think that would work and be better.